llvm/include/llvm/SYCLLowerIR/DeviceConfigFile.td

```diff
@@ -160,9 +160,31 @@ def : TargetInfo<"x86_64", [], [], "", "", 1>;
 
 // TODO: The aspects listed for the intel_gpu targets right now are incomplete; 
 // only the fp16/fp64/atomic64 aspects are listed.
```

Comment:
May be update comment to include SG aspects? Also, is the list of GPUs complete? Thanks

Reply:
I agree about SG sizes comment update.

> Also, is the list of GPUs complete?

Likely no, we have almost 40 lines describing different Intel GPU architectures in #13976, not counting aliases

I think that we need some kind of an integration test here between clang driver, SYCL headers and device config file. All those places list known targets and we want them to be in sync. However, I wouldn't block this commit by lack of such test, but that is something we need to have going forward. The fact that this file is expanded incrementally is expected, I think